packages/ui/src/components/Button/Button.stories.tsx

```diff
@@ -1,30 +1,52 @@
 import React from 'react';
 import { Meta, Story } from '@storybook/react';
 import { Button, ButtonProps } from '.';
+import { Facebook } from '@styled-icons/boxicons-logos/Facebook';
 
 const Template: Story<ButtonProps> = (args) => (
-  <Button {...args}>Call to action</Button>
+  <Button {...args}>Call to action </Button>
 );
 
 export const Primary = Template.bind({});
 
 export const Secondary = Template.bind({});
 Secondary.args = { type: 'secondary' };
 
-export const MediumPrimary = Template.bind({});
-MediumPrimary.args = { size: 'medium' };
+export const XsPrimary = Template.bind({});
+XsPrimary.args = { size: 'xs' };
 
-export const SmallPrimary = Template.bind({});
-SmallPrimary.args = { size: 'small' };
+export const SmPrimary = Template.bind({});
+SmPrimary.args = { size: 'sm' };
 
-export const MediumSecondary = Template.bind({});
-MediumSecondary.args = { type: 'secondary', size: 'medium' };
+export const LgPrimary = Template.bind({});
+LgPrimary.args = { size: 'lg' };
 
-export const SmallSecondary = Template.bind({});
-SmallSecondary.args = { type: 'secondary', size: 'small' };
+export const XsSecondary = Template.bind({});
+XsSecondary.args = { type: 'secondary', size: 'xs' };
+
+export const SmSecondary = Template.bind({});
+SmSecondary.args = { type: 'secondary', size: 'sm' };
+
+export const LgSecondary = Template.bind({});
+LgSecondary.args = { type: 'secondary', size: 'lg' };
+
+export const WithIcon = Template.bind({});
+WithIcon.args = {
+  icon: <Facebook size={20} />,
+  style: {
+    width: '220px',
```

Comment:
Can we remove this from here and instead have a decorator on the export default? Something like the following
```
decorators: [
    (Story: any) => (
      <Container mt="40px">
        <Story />
      </Container>
    ),
  ],
```

Also can we have stories for each one of the different sizes we are having?

Perhaps we can do something like this for all the stories we are having, what you think?
![image](https://user-images.githubusercontent.com/4181674/110215774-730e0600-7ea3-11eb-8f2c-7db40d1745f0.png)

Reply:
![image](https://user-images.githubusercontent.com/58083933/110224247-27c31a00-7eda-11eb-95a9-c9f940d00209.png)

Yes this is very good, it is a lot better.

Reply:
> Can we remove this from here and instead have a decorator on the export default? Something like the following

Hey man, I am confused about something, what is **this** in that sentence? What I see is line 34 to 37 of the file and if I check Github doesn't say more.

```
WithIcon.args = {
  icon: <Facebook size={20} />,
  style: {
    width: '220px',
```
This is what I see in green.

Reply:
The code changed and changed my selection. One sec.

Reply:
I already did it, but I will merge and open a new PR. I want to follow was is best for you/us!